src/main/osd/osd.c

```diff
@@ -1243,6 +1245,10 @@ STATIC_UNIT_TESTED bool osdProcessStats1(timeUs_t currentTimeUs)
         timeUs_t deltaT = currentTimeUs - lastTimeUs;
         osdFlyTime += deltaT;
         stats.armed_time += deltaT;
+#ifdef USE_LAUNCH_CONTROL
+        if (!isLaunchControlActive())
+            osdLaunchTime += deltaT;
```

Comment:
You probably want the counter to reset on consecutive launches
```suggestion
        if (!isLaunchControlActive()) {
            osdLaunchTime += deltaT;
        } else {
            osdLaunchTime = 0;
        }
```